DoSomething/intertwine
# Preload Science Sleuth to deployed intertwine

#### What's this PR do?

Adds the Science Sleuth 2014 to the repo and pre-loads the game when a user navigates to the home page. 

There were also a few changes in `exportsms.js` which somehow didn't get saved when #41 got merged in, they were also made. 
#### Where should the reviewer start?

Check out `preload.js`. 
#### How should this be manually tested?

When we run our local version of Intertwine, it doesn't spin up a server and instead serves static assets from our filesystem. `preload.js` makes ajax requests to retrieve `sciencesleuth2014.html`, which chrome doesn't allow if we retrieve it from the filesystem. (Chrome thinks that it's a cross-site HTTP request, which isn't secure.) 

Hence, in order to test this on your local, you need to spin up the intertwine directory within a local server. An easy way to do this is with the python client. If you have python installed, go into your intertwine directory, and run: 
`python -m SimpleHTTPServer 8888`
Then, you can direct your browser to `localhost:8888` and check to make sure that a `Science_Sleuth_2014` game shows up.
